test/Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -229,7 +228,7 @@ spec_with name create_new_datetime parse_datetime nanoseconds_loss_in_precision=
             time . day . should_equal 1
 
         Test.specify "should add time interval" <|
-            time = create_new_datetime 1970 (zone = Time_Zone.utc) + (Duration.nanoseconds 1)
+            time = create_new_datetime 1970 (zone = Time_Zone.utc) + (Duration.new nanoseconds=1)
```

Comment:
Only now I noticed that we don't seem to have tests for the `millisecond` and `microsecond` parameters of `Date_Time.new`. Sorry that I didn't notice that in the original PR.

Can we add it now or in the next iteration? IMO we should have it tested, at least some very very basic test.